### DISCUSSION TITLE
Send iSpindel data to several services using Node-RED

### DISCUSSION THREAD

**Discussion by jonathanschneider** [BernhardSchlegel/BierBot-Bricks#20] [Show and tell]
Hi all,

I wanted to send data from iSpindel to both Brewfather and BierBot Bricks. So I set up the iSpindel to send data via HTTP to Node-RED which then forwards it to Brewfather and BierBot Bricks.

The flow can be found here:
https://gist.github.com/jonathanschneider/99d24e0cbc336ca1cd53a8de4919fa44

Just insert your Brewfather token in the URL in the "Brewfather" node and your BierBot Bricks API key in the "Format" node and you're good to go!

**Comment by BernhardSchlegel** [comment]
How cool is that ! Thanks for your contribution :)

**Comment by concex17** [comment]
hi, good project. but it is possible to read the ispindel data (angle, temperature ..) on nodered dashboard?

**Reply by BernhardSchlegel** [reply]
Yes, see here https://docs.bierbot.com/hardware/hardware/ispindel

**Reply by concex17** [reply]
sorry, but how i can see the ispindel value on nodered dashboard through bierbot?

**Reply by BernhardSchlegel** [reply]
did you read the link?

**Reply by jonathanschneider** [reply]
BierBot does not send data to Node-RED. If you want to use the iSpindel data in Node-RED you will have to setup the iSpindel to send data to Node-RED and setup Node-RED to forward the data to BierBot.

iSpindel --> Node-RED --> BierBot

**Reply by BernhardSchlegel** [reply]
thanks @jonathanschneider for jumping in! Didn't get the question. If you have a step-by-step guide I'd be highly interested, also for publishing on the blog.
